openbb_terminal/reports/reports_controller.py

```diff
@@ -145,9 +144,6 @@ def call_crypto(self, other_args: List[str]):
 
     @log_start_end(log=logger)
     def call_forecast(self, other_args: List[str]):
-        if is_packaged_application():
```

Comment:
no need to have this check anymore?

Reply:
it's disabling forecast

Reply:
Negatory. We had to add that in to disable the forecast report on rc1

Reply:
Forecast working right now?

Reply:
@Chavithra Yessirrrr. All reports are now thankfully